### ISSUE TITLE
Add support for autofilling addresses and payment information

### ISSUE THREAD

**Issue by influx3k** [mozilla-mobile/firefox-ios#7695]
### Why/User Benefit/User Problem
Come on, it’s 2020.

**Comment by acdha** [comment]
I stopped using Firefox for iOS because of this. None of the work they’re doing matters as much as not having autofill.

**Comment by influx3k** [comment]
Yeah, I agree. It’s 2021 now. Support for this is beyond optional; it’s required to even be considered in the game.

--
Todd Benivegna // todd@benivegna.com

**Comment by karlcow** [comment]
Would be probably interesting to know why they decided to block Firefox on ios
https://github.com/webcompat/web-bugs/issues/71177

**Comment by peanutlasko** [comment]
Any update on this feature? Having address/CC info auto-fill on mobile is only thing stopping me from switching to FF full time.

**Comment by dnarcese** [comment]
Hello! We are currently working on the ability to autofill your payment information- it should be available sometime in April!

**Comment by influx3k** [comment]
Fantastic.  That is good news.
 
--
Todd Benivegna // ***@***.*** ***@***.***

**Comment by peanutlasko** [comment]
@dnarcese is this feature still on track?

**Comment by erikbohnsack** [comment]
@dnarcese 

Hi Daniella! 

Would love to hear about an update on this issue. Crazy that it only has 3 upvotes here, as I think it creates annoyance on a weekly basis for regular users. 

Best, 
Erik

**Comment by dnarcese** [comment]
@afurlan-firefox

**Comment by peanutlasko** [comment]
What does this smiley face mean @dnarcese? Any update on this feature request?
